docs/side_quests/splitting-and-grouping.md

```diff
@@ -0,0 +1,1314 @@
+# Splitting and Grouping
+
+Nextflow helps you work with your data in flexible ways. One of the most useful things you can do is split your data into different streams and then group related items back together. This capability is particularly valuable in bioinformatics workflows where you often need to process different sample types separately before combining results for comparison or joint analysis.
+
+Think of it like sorting mail: you might first separate letters by their destination, process each pile differently, and then recombine items going to the same person. In Nextflow, we use special operators to do this with our scientific data.
+
+Nextflow's channel system is at the heart of this flexibility. Channels connect different parts of your workflow, allowing data to flow through your analysis. You can create multiple channels from a single data source, process each channel differently, and then merge channels back together when needed. This approach lets you design workflows that naturally mirror the branching and converging paths of complex bioinformatics analyses.
+
+In this side quest, we'll explore how to split and group data using Nextflow's powerful channel operators. We'll start with a samplesheet containing information about different samples and their associated data. By the end of this side quest, you'll be able to manipulate and combine data streams effectively, making your workflows more efficient and easier to understand.
+
+You will:
+
+- Read data from files using `splitCsv`
+- Filter and transform data with `filter` and `map`
+- Combine related data using `join` and `groupTuple`
+
+These skills will help you build workflows that can handle multiple samples and different types of data efficiently.
+
+---
+
+## 0. Warmup
+
+### 0.1 Prerequisites
+
+Before taking on this side quest you should:
+
+- Complete the [Hello Nextflow](../hello_nextflow/README.md) tutorial
+- Understand basic Nextflow concepts (processes, channels, operators)
+
+### 0.2 Starting Point
+
+Let's move into the project directory.
+
+```bash
+cd side-quests/splitting-and-grouping
```

Comment:
This directory doesn't exist in this PR. Very likely you already know this Adam and are just waiting to finalize the structure before committing the bams, but I just wanted to check.